app/models/firmware_registry_simple.rb

```diff
@@ -0,0 +1,59 @@
+class FirmwareRegistrySimple < FirmwareRegistry
```

Comment:
Hm wonder if we could come up with something more descriptive than `Simple`...maybe `Http` or `HttpFileServer`?

Reply:
That is essentially all this registry type is right?  Just an http file server with authentication?

Reply:
Yeah, kind of: it's http server that servers file metadata; the actual files may be stored separately (on some external ftp or samba or something)

Reply:
Okay, I ended up with a small refactoring. Now we have 

```
firmware_registry
  |- rest_api_depot.rb   # actual STI-driven implementation
firmware_registry.rb     # base class
```

I think

```
class FirmwareRegistry::RestApiDepot
```
 is better than HttpFileServer because we emphasise the metadata part. Thanks for kicking my ass, I didn't like Simple either because it tells literally nothing.